### DISCUSSION TITLE
Practical Series 4 - Breakout room 12

### DISCUSSION THREAD

**Discussion by DanniAnderson** [IBAHCM/RPiR#267] [Archive]
**Breakout Room 12**

David Anderson - DavidAnderson102

Holly Niven - hollyisbella

Annabel Locke - annabellocke

**Comment by AnnabelLocke** [comment]
my repo; https://github.com/IBAHCM/annabellockeBCI

**Comment by DanniAnderson** [comment]
my repo : https://github.com/IBAHCM/DavidAnderson102BCI.git

**Comment by hollyisbella** [comment]
https://github.com/IBAHCM/hollyisbellaBCI
